docs/_config.yml

```diff
@@ -19,8 +19,8 @@ include:
 
 # These allow the documentation to be updated with newer releases
 # of Spark, Scala, and Mesos.
-SPARK_VERSION: 3.4.0-SNAPSHOT
-SPARK_VERSION_SHORT: 3.4.0
+SPARK_VERSION: 3.5.0-SNAPSHOT
+SPARK_VERSION_SHORT: 3.5.0
 SCALA_BINARY_VERSION: "2.12"
 SCALA_VERSION: "2.12.16"
```

Comment:
When should `SCALA_VERSION`  be changed? The current master uses `2.12.17` instead of `2.12.16`. Maybe the branch-3.4 also needs to be changed?

Reply:
Oh, right. It seems outdated.

Reply:
fixed in https://github.com/apache/spark/pull/39736

Reply:
Thanks. #39736 is merged now.

Reply:
Thank you both!